### ISSUE TITLE
Accessibility: Keyboard: Focus is not clearly visible on I Accept button

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3088]
![image](https://user-images.githubusercontent.com/1005609/78628849-565bef00-784a-11ea-8fbd-7cbeace102e4.png)

**Actual**
The focus is not clearly visible when “I Accept” button gets focus. 

**Expected**
A thick focus rectangle is required for “I Accept” response button similar to other response buttons (see screenshot on right). 

**WCAG Violation**
SC 2.4.7:  Focus Visible
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

SC 3.2.4 : Consistent Identification
https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by compulim** [comment]
This is the styling of Adaptive Cards. Please contact their team for improving focus indicator.

**Comment by compulim** [comment]
This is fixed in v4.

![image](https://user-images.githubusercontent.com/1622400/79022901-da0c2900-7b33-11ea-9e08-e62519e217dc.png)
